src/Network/GRPC/MQTT/Logging.hs

```diff
@@ -23,14 +23,14 @@ data Verbosity
 noLogging :: Logger
 noLogging = Logger (\_ -> pure ()) Silent
 
-logErr :: Logger -> Text -> IO ()
+logErr :: (MonadIO m) => Logger -> Text -> m ()
```

Comment:
```suggestion
logErr :: (MonadIO io) => Logger -> Text -> io ()
```

... I don't always remember to do this and it's just a nit (so feel free to ignore), but sometimes it's just a tiny bit nicer to read when your only monad constraint is `MonadIO`.

Reply:
I like it. I'll add that in.